dashboard/src/contexts/WhoAmIContext.tsx

```diff
@@ -23,7 +23,7 @@ export const WhoAmIContext: FC<PropsWithChildren<WhoAmIContextProps>> = ({ child
     if (!tenantId) {
       setTenant(tenants[0])
     }
-  })
+  }, [])
```

Comment:
good catch as this would run on every render and we only need this to run 1 time

Reply:
I removed this as this was only for setting the cookie.